### ISSUE TITLE
Output the beam of images to logs before convolution

### ISSUE THREAD

**Issue by tjgalvin** [tjgalvin/flint#14]
Some strange errors are coming through in the convolve stage for some sbids. Specifically something like:

```
Encountered exception during execution:
Traceback (most recent call last):
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/prefect/engine.py", line 1719, in orchestrate_task_run
    result = await call.aresult()
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/prefect/_internal/concurrency/calls.py", line 292, in aresult
    return await asyncio.wrap_future(self.future)
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/prefect/_internal/concurrency/calls.py", line 316, in _run_sync
    result = self.fn(*self.args, **self.kwargs)
  File "/scratch3/gal16b/packages/flint/flint/prefect/flows/continuum_pipeline.py", line 215, in task_convolve_image
    return convolve_images(
  File "/scratch3/gal16b/packages/flint/flint/convol.py", line 118, in convolve_images
    beamcon_2D.worker(
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/racs_tools/beamcon_2D.py", line 271, in worker
    savefile(
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/racs_tools/beamcon_2D.py", line 191, in savefile
    header = beam.attach_to_header(header)
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/radio_beam/beam.py", line 307, in attach_to_header
    header.update(self.to_header_keywords())
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/astropy/io/fits/header.py", line 1156, in update
    update_from_dict(k, v)
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/astropy/io/fits/header.py", line 1137, in update_from_dict
    card = Card(k, v)
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/astropy/io/fits/card.py", line 201, in __init__
    self.value = value
  File "/scratch3/gal16b/mambaforge/envs/flint/lib/python3.8/site-packages/astropy/io/fits/card.py", line 349, in value
    raise ValueError(
ValueError: Floating point nan values are not allowed in FITS headers.
```

I believe it is because the original image as a beam above the cut off. It would be nice to confirm this as an operator by printing out the beams of each image.

**Comment by tjgalvin** [comment]
I have patched in this: 2f7c30a32172506e2dcb64c69c174d92ed778273

Indeed, this particular SBID was really trash resolution, much larger than the 25 arcsecond cutoff. Will need to consider what to do in this case in terms of reporting and handling errors.

**Comment by tjgalvin** [comment]
I am not too sure what the best way of handling a single bad beam is. Should it be dropped? A question for the others in the team. Closing this issue.
